.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none"
+}
```

Comment:
Most of these are the defaults, but since we had inconsistencies thought it would be helpful to specify formatting specifications in code